doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -178,7 +178,8 @@ Machine learning
 Asset Tracker v2
 ----------------
 
-|no_changes_yet_note|
+* Updated the Wi-Fi configurations to optimize the RAM usage by about 25kB for nRF91 Series DK and 12kB for Thingy:91 X
+  due to the optimizations done in the lower layers since the initial configuration was made.
```

Comment:
```suggestion
* Updated the Wi-Fi configurations to optimize RAM usage for the nRF91 Series DK and Thingy:91 X.
```
Would this be enough? Not sure whether we need to mention the magnitude of the savings or the optimizations in the lower layers.

Reply:
I knew this will raise comments 😄
If I was using our SDK and would have based my app on this sample, I'd like to know the magnitude if it's significant. And 25kB is a lot (over 10% of the entire RAM). Also, as a user of our SDK, I would like to find out why such a big saving was possible in the first place. So, I'm trying to answer those questions here. If we say "there were RAM optimizations", it's almost like not saying anything.
So, if I had to decide, I would at least mention something about the magnitude and then let the user go and find the commit to look for more information if (s)he wants to.

Reply:
Fair enough, however remember the empty space between the number and kB. And maybe there is no need to mention the lower layers stuff.

Reply:
Modified to "25 kB" and removed the reasoning part about lower layers. Users may find that out from the commit messages which is fairly long for this commit/PR.